config/neutron/templates/neutron-openvswitch-vswitchd.sh.j2

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
+sleep 20
```

Comment:
I must be fixed. That was dirty workaround in order to be sure that openvswitch-db-server is working.